mycore-solr/src/main/java/org/mycore/solr/index/file/tika/MCRSimpleTikaMapper.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * This file is part of ***  M y C o R e  ***
+ * See http://www.mycore.de/ for details.
+ *
+ * MyCoRe is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MyCoRe is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MyCoRe.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.mycore.solr.index.file.tika;
+
+import java.nio.file.Path;
+import java.nio.file.attribute.BasicFileAttributes;
+
+import org.apache.solr.common.SolrInputDocument;
+import org.mycore.common.config.annotation.MCRProperty;
+
+import com.google.gson.JsonElement;
```

Comment:
Do not use Gson!

Reply:
We agreed to use Jackson only for new code and move away from GSON completely - somewhat later.

Reply:
changed to jackson